Source/Scene/Cesium3DTileset.js

```diff
@@ -349,6 +329,36 @@ define([
         when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
     }
 
+    function selectTileWithTilesetContent(tiles3D, selectedTiles, tile, fullyVisible, frameState, replace) {
+        // 1) If its children are not loaded, load the subtree it points to and then select its root child
+        // 2) If its children are already loaded, select its (root) child since the geometric error of it is
+        //    same as this tile's
+        var childrenLength = tile.children.length;
+        var contentUrl = tile._header.content.url;
+
+        // If the subtree has already been added and child
+        // content requested, select the child and continue
+        if ((childrenLength > 0) && (tile.numberOfChildrenWithoutContent === 0)) {
+            selectTile(selectedTiles, tile.children[0], fullyVisible, frameState);
+            return;
+        } else if (replace) {
+            // Otherwise, select the parent tile, to avoid showing an empty space
+            // Note that tile.parent must always be defined here --
```

Comment:
In the case of additive replacement (where we render the parent and children), I don't think there is any concern.

In the case of replacement refine (where we render the parent _or_ the children), I see there is an issue where we would not draw the parent and then traverse to the children who's "content" is loaded because the tiles.json is loaded, but the root tile is not (and this could happen recursively I suppose).

Perhaps we add a state:
```javascript
    var Cesium3DTileContentState = {
        UNLOADED : 0,   // Has never been requested
        LOADING : 1,    // Is waiting on a pending request
        PROCESSING : 2, // Request received.  Contents are being processed for rendering.  Depending on the content, it might make its own requests for external data.
        READY : 3,      // Ready to render.
        FAILED : 4      // Request failed.
    };
```
becomes something like
```javascript
    var Cesium3DTileContentState = {
        UNLOADED : 0,          // Has never been requested
        LOADING : 1,           // Is waiting on a pending request
        PROCESSING : 2,        // Request received.  Contents are being processed for rendering.  Depending on the content, it might make its own requests for external data.
        READY_TO_TRAVERSE : 3, // If content.url is a tiles.json, it is loaded, but its root node is not
        READY_TO_RENDER : 4,   // Ready to render (also implies READY_TO_TRAVERSE).
        FAILED : 5             // Request failed.
    };
```